geoserver/pom.xml

```diff
@@ -11,8 +11,8 @@
   <name>GeoServer 2.x root module</name>
   <properties>
     <maven.test.skip>true</maven.test.skip>
-    <gs.version>2.8.2</gs.version>
-    <gt.version>14.2</gt.version>
+    <gs.version>2.8.3</gs.version>
```

Comment:
This should come along with an update of the `geoserver/geoserver-submodule/`, pointing onto the `2.8.3-georchestra` branche.